articles/libraries/auth0js/migration-guide.md

```diff
@@ -0,0 +1,62 @@
+---
+section: libraries
+description: How to migrate from auth0.js v7 to auth0.js v8
+url: /libraries/auth0js/migration-guide
+---
+
+# Auth0.js v7 to v8 Migration Guide
+
+The following instructions assume you are migrating from **auth0.js v7** to **auth0.js v8**. 
+
+The goal of this migration guide is to provide you with all of the information you would need to update Auth0.js in your application. Of course, your first step is to include the latest version of auth0.js. Beyond that, take a careful look at each of the areas on this page. You will need to change your implementation of auth0.js to reflect the new changes. Take a look below for more information!
+
```

Comment:
I'd add a note saying we recommend auth0.js v8 if you have the OAuth2 flag enabled in your tenant